ios/RNCWKWebView.m

```diff
@@ -88,7 +88,20 @@ - (void)didMoveToWindow
         wkWebViewConfig.processPool = [[RNCWKProcessPoolManager sharedManager] sharedProcessPool];
     }
     wkWebViewConfig.userContentController = [WKUserContentController new];
-    [wkWebViewConfig.userContentController addScriptMessageHandler: self name: MessageHanderName];
+
+    if (_messagingEnabled) {
+      [wkWebViewConfig.userContentController addScriptMessageHandler: self name: MessageHandlerName];
+
+      NSString *source = [NSString stringWithFormat:
+        @"window.@ = {"
+         "  postMessage: function (data) {"
+         "    window.webkit.messageHandlers.@.postMessage(String(data));"
+         "  }"
+         "};", MessageHandlerName, MessageHandlerName
+      ];
+      [self injectJavaScript: source];
```

Comment:
This will make sure window.ReactNativeWebview.postMessage is defined, the same way it is defined in Android, which is great!

In here we don't need messageReceived because we can pass messages without changing the location.

So, this all looks fine!